network-api/networkapi/templates/fragments/libraries/author_card.html

```diff
@@ -13,7 +13,7 @@
         {% endwith %}
     </div>
     <div>
-        <div class="tw-h5-heading tw-mb-2 tw-text-blue-80 group-hover:tw-underline">
+        <div class="author-name tw-h5-heading tw-mb-2 tw-text-blue-80 group-hover:tw-underline">
```

Comment:
Updated for testing purposes (to make sure that author names are accurately being rendered in the template)